src/test/java/seedu/address/logic/commands/ListClaimsCommandTest.java

```diff
@@ -0,0 +1,112 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.logic.commands.CommandTestUtil.showClientAtIndex;
+import static seedu.address.testutil.TypicalClients.getTypicalAddressBook;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_CLIENT;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_CLIENT;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.client.Client;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for
+ * {@code ListClaimsCommand}.
+ */
+public class ListClaimsCommandTest {
+
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    @Test
+    public void execute_validIndexUnfilteredList_success() {
+        Client clientToDisplay = model.getFilteredClientList().get(INDEX_FIRST_CLIENT.getZeroBased());
+        ListClaimsCommand listClaimsCommand = new ListClaimsCommand(INDEX_FIRST_CLIENT);
+
+        String expectedMessage = String.format(ListClaimsCommand.MESSAGE_SUCCESS,
+                clientToDisplay.getName().toString(),
+                clientToDisplay.getInsurancePlansManager().accessClaims());
+
+        Model expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+
+        assertCommandSuccess(listClaimsCommand, model, expectedMessage, expectedModel);
+    }
```

Comment:
Maybe some javadoc could be written here and for `validIndexFilteredList` as it's not clear what these tests are testing right now?

Reply:
I agree, I will add javadoc to make it clearer.

Reply:
alright understood!